[2025-01-31T09:58:37.725+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T09:58:37.759+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-python-etl-dag.check scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T09:58:37.790+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-python-etl-dag.check scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-01-31T09:58:37.792+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-31T09:58:38.045+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): check> on 2025-01-30 00:00:00+00:00
[2025-01-31T09:58:38.056+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T09:58:38.059+0000] {standard_task_runner.py:72} INFO - Started process 102 to run task
[2025-01-31T09:58:38.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my-first-python-etl-dag', 'check', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpp9q8df37']
[2025-01-31T09:58:38.062+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask check
[2025-01-31T09:58:38.150+0000] {task_command.py:467} INFO - Running <TaskInstance: my-first-python-etl-dag.check scheduled__2025-01-30T00:00:00+00:00 [running]> on host 0d287206d23f
[2025-01-31T09:58:38.329+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='my-first-python-etl-dag' AIRFLOW_CTX_TASK_ID='check' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-01-31T09:58:38.331+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T09:58:38.332+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T09:58:38.333+0000] {logging_mixin.py:190} INFO - Current task name:check state:running start_date:2025-01-31 09:58:37.760692+00:00
[2025-01-31T09:58:38.334+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag and current dag run status:running
[2025-01-31T09:58:38.335+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T09:58:38.336+0000] {logging_mixin.py:190} INFO - Inside Check
[2025-01-31T09:58:38.338+0000] {logging_mixin.py:190} INFO - root,0,/root
[2025-01-31T09:58:38.339+0000] {logging_mixin.py:190} INFO - daemon,1,/usr/sbin
[2025-01-31T09:58:38.340+0000] {logging_mixin.py:190} INFO - bin,2,/bin
[2025-01-31T09:58:38.340+0000] {logging_mixin.py:190} INFO - sys,3,/dev
[2025-01-31T09:58:38.342+0000] {logging_mixin.py:190} INFO - sync,4,/bin
[2025-01-31T09:58:38.343+0000] {logging_mixin.py:190} INFO - games,5,/usr/games
[2025-01-31T09:58:38.344+0000] {logging_mixin.py:190} INFO - man,6,/var/cache/man
[2025-01-31T09:58:38.345+0000] {logging_mixin.py:190} INFO - lp,7,/var/spool/lpd
[2025-01-31T09:58:38.346+0000] {logging_mixin.py:190} INFO - mail,8,/var/mail
[2025-01-31T09:58:38.347+0000] {logging_mixin.py:190} INFO - news,9,/var/spool/news
[2025-01-31T09:58:38.348+0000] {logging_mixin.py:190} INFO - uucp,10,/var/spool/uucp
[2025-01-31T09:58:38.349+0000] {logging_mixin.py:190} INFO - proxy,13,/bin
[2025-01-31T09:58:38.350+0000] {logging_mixin.py:190} INFO - www-data,33,/var/www
[2025-01-31T09:58:38.351+0000] {logging_mixin.py:190} INFO - backup,34,/var/backups
[2025-01-31T09:58:38.352+0000] {logging_mixin.py:190} INFO - list,38,/var/list
[2025-01-31T09:58:38.352+0000] {logging_mixin.py:190} INFO - irc,39,/run/ircd
[2025-01-31T09:58:38.353+0000] {logging_mixin.py:190} INFO - _apt,42,/nonexistent
[2025-01-31T09:58:38.354+0000] {logging_mixin.py:190} INFO - nobody,65534,/nonexistent
[2025-01-31T09:58:38.356+0000] {logging_mixin.py:190} INFO - ***,50000,/home/***
[2025-01-31T09:58:38.357+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T09:58:38.374+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T09:58:38.375+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my-first-python-etl-dag, task_id=check, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250131T095837, end_date=20250131T095838
[2025-01-31T09:58:38.393+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T09:58:38.394+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T09:58:38.395+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag queued_at:2025-01-31 09:58:15.887300+00:00
[2025-01-31T09:58:38.396+0000] {logging_mixin.py:190} INFO - Task hostname:0d287206d23f operator:PythonOperator
[2025-01-31T09:58:38.438+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T09:58:38.466+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-31T09:58:38.470+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
